docs/developers-guide/decorators/json-schema-enabled decorator.md

```diff
@@ -0,0 +1,93 @@
+# The json-schema-enabled Decorator
+This document explains how the json-schema-enabled decorator works and how it is applied to:
+* values
+* types
+* modules
+It also explains how a Json Schema is generated based on the annotation
+
+## Overview of json-schema-enabled Decorator
+The json-schema-enabled decorator is modeled as a boolean type in the JsonSchema module in the
+Attribute model.
+When a json-schema-enabled decorator value of true is applied to a type or module, then json-schema
+backend code generation would be based on these. 
+When applied to a type or value, the fully qualified name is used as the key in the attributed dictionary.
+But when applied to a module, the module name is used as the key
+All decorator configurations for a project are place in the attributes-conf.json file placed next to the main IR.
```

Comment:
This comment in itself is confusing. You'll need to at least have a pointer to another doc that explains how decorators are set up or maybe include it right here.

Reply:
This document details these in the subsequent section. I intend this section as an overview